### PR TITLE
Add a --skip mode for visual diff tests and make them PR blocking

### LINKED ISSUE #10164: Figure out how to make Percy tests PR blocking, and yet, not have to run the tests for all PRs
Discovered via #10159

The PR remained permanently blocked because Percy tests are now "required", but we don't run visual tests for validator changes.

### PR DESCRIPTION
**Pull request by rsimha-amp**
Fixes #10164

### CODE REVIEW

**Comment by rsimha-amp** [comment]
/to @cramforce or @choumx for review, after which percy can be made a required check on Github.

File: build-system/pr-check.js [hunk 1]
```diff
@@ -396,6 +400,9 @@ function main(argv) {
       if (buildTargets.has('RUNTIME')) {
         command.runUnitTests();
       }
+    } else {
+      // Generates a blank Percy build to satisfy the required Github check.
+      command.runVisualDiffTests(true /*pass*/);
```
Comment (choumx) [inline build-system/pr-check.js]: `command.runVisualDiffTests(/* opt_pass */ true);`

File: build-system/pr-check.js [hunk 2]
```diff
@@ -280,9 +280,13 @@ const command = {
     timedExecOrDie(
         `${gulp} test --nobuild --saucelabs --integration --compiled`);
   },
-  runVisualDiffTests: function() {
+  runVisualDiffTests: function(opt_pass) {
```
Comment (choumx) [inline build-system/pr-check.js]: `opt_skip`?

Comment (rsimha-amp) [inline build-system/pr-check.js]: Done.

File: build-system/pr-check.js (hunk 1 above)
Comment (rsimha-amp) [inline build-system/pr-check.js]: Done.

**Pull request merged**